examples/tests/sdk_tests/bucket/signedUrl.w

```diff
@@ -0,0 +1,14 @@
+bring cloud;
+bring http;
+bring util;
+
+let testBucket = new cloud.Bucket(public: true) as "testBucket";
+
+test "signedUrl" {
+  let var error = "";
+  testBucket.put("file1.txt", "Foo");
+
+  let signedUrl = testBucket.signedUrl("file1.txt");
+  assert(signedUrl != "");
```

Comment:
Can you test the exact value instead of making sure it is not empty?

Reply:
@revitalbarletz Sure I can do that